src/components/Main/Teams/DroppableTeamMemberList.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import {
+  Droppable,
+  Draggable,
+  DraggableProvided,
+  DroppableProvided,
+} from 'react-beautiful-dnd';
+import { MembersData, DraggedItem } from './models';
+import { v4 as uuid } from 'uuid';
+
+// Material UI Components
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import { TextField } from '@material-ui/core';
+import { TeamMemberRow } from './TeamMemberRow';
+
+interface DroppableTeamMembersListProps {
+  role: string;
+  members: MembersData[];
+  canDrop: () => boolean;
+  draggedItem: DraggedItem;
+}
+
+export const DroppableTeamMembersList = ({
+  role,
+  members,
+  canDrop,
+  draggedItem,
+}: DroppableTeamMembersListProps) => {
+  const classes = useStyles();
+
+  const [memberInput, setMemberInput] = useState({ value: '', error: '' });
+
+  // handleSubmit and handleDelete are placeholder functions
+  function handleSubmit(event: any) {
+    event.preventDefault();
+    // useSelector if name doesn't exist in reducer, return invalid
+    if (members.find((member: any) => member.name === memberInput.value)) {
+      setMemberInput({ ...memberInput, error: 'User already exists' });
+    } else {
+      members.push({ id: uuid(), name: memberInput.value });
+      setMemberInput({ ...memberInput, value: '', error: '' });
+    }
+  }
+
+  function handleDelete(selectedMember: any, role: any, index: any) {
+    if (members.find((member) => member.name === selectedMember.name)) {
+      members.splice(index, 1);
+      setMemberInput({ ...memberInput, value: '', error: '' });
+    }
+  }
+
+  return (
+    <Droppable droppableId={role} key={role} isDropDisabled={canDrop()}>
+      {(provided: DroppableProvided) => (
+        <List dense ref={provided.innerRef} className={classes.list}>
+          {members.map((member: MembersData, index: number) => {
+            return draggedItem.source === '' || !(draggedItem.source === 'USERBANK') ? (
```

Comment:
so is this ternary saying "iff a dragged item is coming from another team, then make the team member draggable"?

Reply:
thanks! this is a great question. actually the only condition this cares for is as long as the source does NOT come from the sidebar. so I'm removing the first condition